templates/taxi/car_detail.html

```diff
@@ -19,4 +19,14 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+
+  {% if user not in car.drivers.all %}
```

Comment:
Ensure that the `car` object passed to the template includes a `drivers` attribute that is a queryset of driver objects. This is necessary for the conditional logic to determine if the user is already assigned to the car.